src/controllers/sheets.ts

```diff
@@ -1,7 +1,9 @@
 import { getSheetData } from '../utils/gsuite/sheets';
 import { Applicant } from '../types/applicant';
+import ApplicantDB, { IApplicant } from '../models/applicant';
```

Comment:
nit: general practice is this is just imported as Applicant not ApplicantDB because the whole MongoDB instance is the DB while Applicant is just a model in the db.

Reply:
Yeah, its just the Applicant model, but since I'm importing the Applicant interface from types/applicant, I had to rename it to something else, so I just added the 'DB' to differentiate this as the Applicant from models/applicant

Reply:
Ok I see that now, for now lets change to ApplicantModel - we should figure out how we can compress IApplicant in the Model and the Applicant type into one thing later on. It seems like a lot of duplicated functionality to essentially do the same thing. I wonder what the best practice for something like that is.